dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -90,7 +90,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
                 AppModel::Identity::IsPackagedProcess() ? registeredClsid : winrt::guid(storedComActivatorString),
                 winrt::make<AppNotificationActivationCallbackFactory>().get(),
                 CLSCTX_LOCAL_SERVER,
-                REGCLS_MULTIPLEUSE,
+                m_notificationHandlers ? REGCLS_MULTIPLEUSE : REGCLS_SINGLEUSE,
```

Comment:
Add a comment here along the lines of:
// If the developer has registered event handlers, COM will ensure that all activators are routed inproc else COM will launch a new process of the win32 app for subsequent invocations